### PR TITLE
DAOS-6442 test: Disable daos_test rebuild 18

### PR DESCRIPTION
**Pull request by sylviachanoiyee**

### CODE REVIEW

File: src/tests/ftest/daos_test/daos_core_test-rebuild.py [hunk 1]
```diff
@@ -32,6 +32,9 @@ class DaosCoreTestRebuild(DaosCoreBase):
     :avocado: recursive
     """
 
+    # Test variants that should be skipped
+    CANCEL_FOR_TICKET = [["DAOS-6442", "test_name", "rebuild tests 18"]]
```
Comment (phender) [inline src/tests/ftest/daos_test/daos_core_test-rebuild.py:36]: While we're making changes can we also fix https://github.com/daos-stack/daos/blob/master/src/tests/ftest/daos_test/daos_core_test-rebuild.yaml#L60:
```suggestion
    CANCEL_FOR_TICKET = [
        ["DAOS-5851", "test_name", "rebuild tests 0-10"]
        ["DAOS-6442", "test_name", "rebuild tests 18"]
    ]
```
+ uncomment the `daos_test/daos_core_test-rebuild.yaml` variant?

**Review by phender — CHANGES_REQUESTED** [review]

File: src/tests/ftest/daos_test/daos_core_test-rebuild.py [hunk 2]
```diff
@@ -32,6 +32,12 @@ class DaosCoreTestRebuild(DaosCoreBase):
     :avocado: recursive
     """
 
+    # Test variants that should be skipped
+    CANCEL_FOR_TICKET = [
+        ["DAOS-5851", "test_name", "rebuild tests 0-10"]
+        ["DAOS-6442", "test_name", "rebuild tests 18"]
```
[the remark this answers is not in the document]
Comment (phender) [reply src/tests/ftest/daos_test/daos_core_test-rebuild.py:38]: Sorry I forgot the comma in my suggestion.  It should be:
```suggestion
        ["DAOS-5851", "test_name", "rebuild tests 0-10"],
        ["DAOS-6442", "test_name", "rebuild tests 18"]
```

**Review by mjmac — APPROVED** [review]

**Review by frostedcmos — APPROVED** [review]

File: src/tests/ftest/daos_test/daos_core_test-rebuild.py [hunk 3]
```diff
@@ -32,6 +32,12 @@ class DaosCoreTestRebuild(DaosCoreBase):
     :avocado: recursive
     """
 
+    # Test variants that should be skipped
+    CANCEL_FOR_TICKET = [
+        ["DAOS-5851", "test_name", "rebuild tests 0-10"],
```
Comment (phender) [inline src/tests/ftest/daos_test/daos_core_test-rebuild.py:37]: This is missing the corresponding change to uncomment the following https://github.com/daos-stack/daos/blob/master/src/tests/ftest/daos_test/daos_core_test-rebuild.yaml#L60-L65.  Without it we will not see a CANCEL state for the Rebuild 0-10 test varaint.

**Review by phender — CHANGES_REQUESTED** [review]

File: src/tests/ftest/daos_test/daos_core_test-rebuild.py (hunk 3 above)
Comment (sylviachanoiyee) [reply src/tests/ftest/daos_test/daos_core_test-rebuild.py:37]: yes, let me do that in a separate PR. Let's not delay the disabling of rebuild18.

**Review by phender — APPROVED** [review]
Changes to daos_core_test-rebuild.py will be handled in a separate PR.

**Pull request merged**